### PR TITLE
Hugo scaffolding

### PR DESCRIPTION
**Pull request by Andertech**
Started working on the Hugo scaffold of the new site, WIP.

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+metadataformat: "yaml"
+languagecode: "en-us"
+baseurl: "http://www.vividcortex.com/docs"
```
Comment (mtrias) [inline config.yaml]: Can we avoid hardcoding the protocol here?

Comment (Andertech) [inline config.yaml]: Pretty sure we can, good point :+1:

**Comment by Andertech** [comment]
@mtrias @nachocaru I'm merging this, I will open smaller PRs to address smaller issues.

File: config.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+pluralizelisttitles: false
+paginate: 25
+paginatePath: "page"
+blackfriday:
+  plainIdAnchors: true
+params:
+  env: "dev"
+  AuthorName: "VividCortex"
+  GitHubUser: "VividCortex"
+  GTMID: "6NVF"
```
Comment (mtrias) [inline config.yaml]: This needs to be changed

File: config.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+---
+metadataformat: "yaml"
+languagecode: "en-us"
+baseurl: "http://vividcortex.com/"
```
Comment (mtrias) [inline config.yaml]: this should be docs.vividcortex.com

**Pull request merged**